org.abapgit.adt.ui/src/org/abapgit/adt/ui/internal/dialogs/AbapGitStagingCredentialsDialog.java

```diff
@@ -77,11 +75,6 @@ protected Control createDialogArea(Composite parent) {
 		data = new GridData(GridData.FILL_HORIZONTAL);
 		this.passwordField.setLayoutData(data);
 
-		//Set credentials in the dialog if retrieved from Secure Storage
-		if (this.repoCredentials != null) {
-			this.usernameField.setText(this.repoCredentials.getUser());
-			this.passwordField.setText(this.repoCredentials.getPassword());
-		}
```

Comment:
We no longer set the credentials retrieved from Secure Storage in the credential dialog and also the credential dialog is not shown to the user in case credentials were already present in Secure Storage. All operations in staging view requiring credentials are directly performed after retrieving creds from Secure Storage.